Describe workaround for Linux csum offload issue (#101)
Juniper/contrail-windows-docs: doc/Known_issues/Known_issues.md

![CredSSP encryption Oracle remediation](CredSSPError.png)

If you see the above error, please refer to [CredSSP encryption Oracle remediation bestiary](CredSSP_error_bestiary.md)

> ### I cannot ping Windows container from Linux VM

Reviewer: ```suggestion
### I cannot ping container on Windows from VM on Linux
```

> ### I cannot ping Windows container from Linux VM on OpenStack compute node

Reviewer: doesn't have to be openstack I think

> Issue is related to checksum handling on the Linux Tungsten Fabric compute node.

Reviewer: ```suggestion
This issue is related to checksum handling on the Linux Tungsten Fabric compute node.
```

> Issue is related to checksum handling on the Linux Contrail compute node.

Reviewer: please refrain from using the name Contrail

If checksum offload is enabled, Linux transmits packets with inner IP checksums not calculated correctly.

> When Windows compute nodes receive such encapsulated packets, after vRouter decapsulates them, receiving container's TCP/IP stack drops a packet due to incorrect checksum.

Reviewer: First you talk about packets and then about "a packet"
```suggestion
When Windows compute nodes receive such encapsulated packets, after vRouter decapsulates them, receiving container's TCP/IP stack drops the packets due to incorrect checksum.
```

> To workaround this behaviour, you can disable checksum offloading on Linux compute node.

Reviewer: `workaround` is a noun.
```suggestion
To work around this behaviour, you can disable checksum offloading on Linux compute node.
```
